<launch>
  <node pkg="pan_tilt_driver" type="pan_tilt_driver_node" name="pan_tilt_driver_node" args="-p tcp:192.168.1.10" output="screen" >
    <param name="simulation" type="bool" value="false" />
  </node>
</launch>
